cmd/composectl/cmd/check.go

```diff
@@ -266,7 +267,13 @@ func checkIfInstalled(ctx context.Context, appRefs []string, srcStorePath string
 				if s, err := reference.Parse(imageUri); err == nil {
 					taggedUri := s.Locator + ":" + (s.Digest().Encoded())[:7]
```

Comment:
what's this `[:7]` doing? Is this a "short-hash"? If so, is it always 7 long? We've hit edge cases in Git where the short-hash isn't always the same length

Reply:
It is always `7` because we set it during app image installation. It occurs in aklite for the versions <= v93 (https://github.com/foundriesio/aktualizr-lite/blob/b4634e45afbf37cc8c11b043dabbc8348dccc982/src/docker/restorableappengine.cc#L565) (setting image tag in the skopeo copy command which loads image to the docker store).
And it occurs in `composectl install` https://github.com/foundriesio/composeapp/blob/c62494f56b123a411f7221f4ee87ed839d0930bd/pkg/compose/v1/install.go#L42. for versions >= v94.